tests/components/radarr/test_init.py

```diff
@@ -1,16 +1,18 @@
 """Test Radarr integration."""
 from aiopyarr import exceptions
+import pytest
 
 from homeassistant.components.radarr.const import DEFAULT_NAME, DOMAIN
 from homeassistant.config_entries import ConfigEntryState
 from homeassistant.core import HomeAssistant
 from homeassistant.helpers import device_registry as dr
 
-from . import create_entry, patch_radarr, setup_integration
+from . import create_entry, mock_connection, patch_radarr, setup_integration
 
 from tests.test_util.aiohttp import AiohttpClientMocker
 
 
+@pytest.mark.freeze_time("2021-12-03 00:00:00+00:00")
```

Comment:
Want to send these misc cleanups in another PR? They seem separate form the calendar entity if i understand right

Reply:
This has to do with the while statement. This is to prevent the tests from hanging. If I drop it, then this can be removed.